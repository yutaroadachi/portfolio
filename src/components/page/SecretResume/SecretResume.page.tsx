import React, { ComponentProps } from 'react'
import { Layout } from 'src/components/ui/Layout'
import { MetaTag } from 'src/components/ui/MetaTag'
import { SecretResume } from './SecretResume'

export type SecretResumePageProps = ComponentProps<typeof SecretResume>

export const SecretResumePage = ({}: SecretResumePageProps) => {
  return (
    <Layout>
      <MetaTag
        title="職務経歴書"
        description="Yutaro Adachi(ソフトウェアエンジニア)の職務経歴書です。"
        noindex
      />
      <SecretResume />
    </Layout>
  )
}
