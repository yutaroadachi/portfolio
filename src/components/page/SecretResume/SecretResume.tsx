import { Box, chakra, Stack } from '@chakra-ui/react'
import React from 'react'

export type SecretResumeProps = {}

export const SecretResume = ({}: SecretResumeProps) => {
  return (
    <Box>
      <chakra.h1 id="secret-resume" textStyle="h1" color="main" mb={8}>
        職務経歴書
      </chakra.h1>
      <Stack spacing={8}>
        <Box>準備中</Box>
      </Stack>
    </Box>
  )
}
