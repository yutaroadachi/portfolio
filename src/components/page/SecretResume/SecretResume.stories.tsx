import { Meta, Story } from '@storybook/react'
import React from 'react'
import { SecretResume, SecretResumeProps } from './SecretResume'

export default {
  title: 'page/SecretResume',
  component: SecretResume,
} as Meta<SecretResumeProps>

type Template = Story<SecretResumeProps>

const Template: Template = (args) => <SecretResume {...args} />

export const Default: Template = Template.bind({})
