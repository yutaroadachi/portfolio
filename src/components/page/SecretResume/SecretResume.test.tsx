import React from 'react'
import { render } from 'src/__tests__/utils'
import { SecretResume } from './SecretResume'

test('SecretResume', () => {
  const { container } = render(<SecretResume />)
  expect(container).toMatchSnapshot()
})
