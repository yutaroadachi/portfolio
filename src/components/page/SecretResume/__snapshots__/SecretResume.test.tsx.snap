// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecretResume 1`] = `
<div>
  <div
    class="css-0"
  >
    <h1
      class="css-15mzpqr"
      id="secret-resume"
    >
      職務経歴書
    </h1>
    <div
      class="chakra-stack css-aajx3r"
    >
      <div
        class="css-0"
      >
        準備中
      </div>
    </div>
  </div>
</div>
`;
